Detectors/MUON/MCH/Workflow/src/PreClusterFinderSpec.cxx

```diff
@@ -32,7 +32,7 @@
 
 #include "MCHBase/Digit.h"
 #include "MCHBase/PreCluster.h"
-#include "PreClusterFinder.h"
+#include "../../PreClustering/src/PreClusterFinder.h"
```

Comment:
No, that's not the correct way to reference includes from another target. If you need the PreClusterFinder.h and it's not currently exported by the MCHPreClustering target, then that must be changed. @pillot any objection to export that file ?

Reply:
@pillot Actually I'm starting to look at this and it might requires a bit of work if we don't want to expose the PreClusterFinderMapping (and I guess we don't want to).

Reply:
@pillot I made an attempt to "hide" the PreClusterFinderMapping while still exporting the PreClusterFinder.h, see https://github.com/AliceMCH/AliceO2/compare/mch-dpl-workflows...aphecetche:expose-preclusterfinder?expand=1 
I've just tried the compilation stage though (i.e. the easy part ;-) ), as the (closure) tests we've already discussed about are not currently in AliceO2, so it'd be good if you could try and see if I broke everything or not with this "structural" change...

Reply:
Thanks for the solution! It looks fine but I will run some tests next week.

Reply:
What should we do for the moment? I agree that the above include statement is an ugly workaround...